changelog/v1.18.0-beta22/tls-secret-validation.yaml

```
@@ -0,0 +1,9 @@
+changelog:
+  - type: FIX
+    issueLink: https://github.com/solo-io/solo-projects/issues/6772
+    resolvesIssue: false
+    description: >-
+      Plugs a gap where go would check a secret for validity per spec but envoy is more aggressive.
+      For example a tls secret with a certchain that contains an invalid pem block will be rejected by envoy but not go.
+      Prior to this pr these types of secrets would be accepted by gloo and nacked by envoy.
```

nit: I'm not sure if we have specific team standards around this, but since it's customer facing we should make sure changelog messages have correct capitalization for acronyms and proper nouns

```suggestion
      Plugs a gap where go would check a secret for validity per spec but Envoy is more aggressive.
      For example a TLS secret with a certChain that contains an invalid PEM block will be rejected by Envoy but not Go.
      Prior to this PR these types of secrets would be accepted by Gloo and nacked by Envoy.
```